docs/faqs.md

```diff
@@ -34,8 +34,13 @@ comments locally, you will need to update the score of your local comments
 manually. Here's how:
 
 1. Open the terminal.
-2. Run `psql PracticalDeveloper_development` to open `psql`, the PostgreSQL
+2. Run `rails dbconsole` to open the PostgreSQL terminal. Alternatively, run
+   `psql PracticalDeveloper_development` to open `psql`, the PostgreSQL
    terminal.
+
+> Note: dbconsole reads database information from config/database.yml which is
+> always better since database configs might change in the future.
```

Comment:
I like the spirit of explaining this, but can we move this down beneath the list of steps? I think the break from the steps is a bit confusing.

What do you think?

Reply:
Totally agreed with breaking the flow from an user perspective. The note stands there to explain why we want users to go to the first alternative, but it didn't strike me that as an user, I can read the note later as well. I'll update this in the next commit.